fast64_internal/z64/actor/operators.py

```diff
@@ -12,7 +12,7 @@ class OOT_SearchChestContentEnumOperator(Operator):
     bl_property = "chest_content"
     bl_options = {"REGISTER", "UNDO"}
 
-    chest_content: EnumProperty(items=game_data.z64.actors.ootEnumChestContent, default="item_heart")
+    chest_content: EnumProperty(items=lambda self, context: game_data.z64.get_enum("chest_content"), default=1)
```

Comment:
Does this work? I thought properties had to have enums which were permanently fixed. What happens if the enum was {A, B, C}, the property was set to B, and then the enum was changed to {D, E, F}? Does the property change to the default value of the new enum? If it keeps the index, what if it was previously C and the enum was changed to {D, E} (out of bounds)?

Reply:
I don't mean if someone edits the XML data but if someone changes the selected game. This could be, import something from OoT, change to MM, export it, or it could be just start Blender in the default scene and change the game. I'm not really worried about correctness, but about Blender not crashing. Anyway just test it.

Reply:
each game got their own enums, only the operator is common iirc so it should be fine, see `get_prop_name` and `create_game_props` (previously `initOOTActorProperties`) in `actor/properties.py`